.github/workflows/ci-base-tests.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           pip install --upgrade pip
           pip install --upgrade wheel
           pip install --upgrade -r requirements.txt
-          pip install --upgrade -e .[train,test,camera-obs]
+          pip install --upgrade -e .[train,test,camera-obs,waymo]
```

Comment:
Is there any `waymo` related tests which requires it to be installed here?

Reply:
No actually we just needed to `requirements.txt` to `scenario/waymo` folder to prevent CI tests to fail when building the waymo scenario. So i added that. Thanks for the reminder.